clang/lib/Sema/TreeTransform.h

```diff
@@ -5490,6 +5490,9 @@ TreeTransform<Derived>::TransformDependentSizedArrayType(TypeLocBuilder &TLB,
   EnterExpressionEvaluationContext Unevaluated(
       SemaRef, Sema::ExpressionEvaluationContext::ConstantEvaluated);
 
+  // VLA bounds are not truly constant.
```

Comment:
This is not always a VLA right? So maybe saying `If we have a VLA then it won't be a constant`